include/deal.II/numerics/vector_tools.h

```diff
@@ -317,61 +319,238 @@ namespace VectorTools
+   * @f[
+   * \int_A f(x) dx \approx \sum_q f(q) \omega_q.
+   * @f]
    */
   enum NormType
   {
     /**
-     * The function or difference of functions is integrated on each cell.
+     * The function or difference of functions is integrated on each cell $K$:
+     * @f[
+     *   E_K
+     * = \int_K \sum_c (\hat{f}_c - f_c) \, w_c
+     * = \int_K \sum_c e_c \, w_c
+     * @f]
+     * and summed up to get
+     * @f[
+     *   E = \sum_K E_K
+     *     = \int_\Omega \sum_c (\hat{f}_c - f_c) \, w_c
+     * @f]
+     * or, for $w \equiv 1$:
+     * @f[
+     *   E = \int_\Omega (\hat{f} - f)
+     *     = \int_\Omega e.
+     * @f]
+     *
+     * Note: This differs from what is typically known as
+     * the mean of a function by a factor of $\frac{1}{|\Omega|}$. To
+     * compute the mean you can also use compute_mean_value(). Finally,
+     * pay attention to the sign: if $\hat{f}=0$, this will compute the
+     * negative of the mean of $f$.
      */
     mean,
+
     /**
-     * The absolute value of the function is integrated.
+     * The absolute value of the function is integrated:
+     * @f[
+     *   E_K = \int_K \sum_c |e_c| \, w_c
+     * @f]
+     * and
+     * @f[
+     *   E = \sum_K E_K = \int_\Omega \sum_c |e_c| w_c,
+     * @f]
+     * or, for $w \equiv 1$:
+     * @f[
+     *   E  = \| e \|_{L^1}.
+     * @f]
      */
     L1_norm,
+
     /**
      * The square of the function is integrated and the the square root of the
-     * result is computed on each cell.
+     * result is computed on each cell:
+     * @f[
+     *   E_K = \sqrt{ \int_K \sum_c e_c^2 \, w_c }
+     * @f]
+     * and
+     * @f[
+     *   E = \sqrt{\sum_K E_K^2} = \sqrt{ \int_\Omega  \sum_c e_c^2 \, w_c }
+     * @f]
+     * or, for $w \equiv 1$:
+     * @f[
+     *   E = \sqrt{ \int_\Omega e^2 }
+     *     = \| e \|_{L^2}
+     * @f]
      */
     L2_norm,
+
     /**
-     * The absolute value to the <i>p</i>th power is integrated and the pth
-     * root is computed on each cell. The exponent <i>p</i> is the last
-     * parameter of the function.
+     * The absolute value to the $p$-th power is integrated and the $p$-th
+     * root is computed on each cell. The exponent $p$ is the @p
+     * exponent argument of integrate_difference() and compute_mean_value():
+     * @f[
+     *   E_K = \left( \int_K \sum_c |e_c|^p \, w_c \right)^{1/p}
+     * @f]
+     * and
+     * @f[
+     *   E = \left( \sum_K E_K^p \right)^{1/p}
+     * @f]
+     * or, for $w \equiv 1$:
+     * @f[
+     *   E = \| e \|_{L^p}.
+     * @f]
      */
     Lp_norm,
+
     /**
-     * The maximum absolute value of the function.
+     * The maximum absolute value of the function:
+     * @f[
+     *   E_K = \sup_K \max_c |e_c| \, w_c
+     * @f]
+     * and
+     * @f[
+     *   E = \max_K E_K
+     * = \sup_\Omega \max_c |e_c| \, w_c
+     * @f]
+     * or, for $w \equiv 1$:
+     * @f[
+     *   E  = \sup_\Omega \|e\|_\infty = \| e \|_{L^\infty}.
+     * @f]
```

Comment:
Can you add a sentence that says that in practice the maximum is not computed as the `\sup`, but as the `\max` over all quadrature points?

(I'm thinking whether there is a point of using `\sup` instead of `\max` in the definition above, but the distinction of course doesn't matter when approximated on a finite set of quadrature points.)